### DISCUSSION TITLE
چند تا ایده دارم

### DISCUSSION THREAD

**Discussion by crowlink** [hiddify/Hiddify-Manager#988] [Ideas ایدهها]
با درود و خسته نباشید خدمت تیم هایدی فای 

چند تا ایده دارم
+اعمال مجدودیت برای مدیرها
+قابلیت جابجایی کاربران بین مدیرها
+ محدودیت تعداد کانکشن های همزمان
--------
اضافه کردن چند پرتکل جدید
SSH/SSL/TLS Tunneling 

در کنارش به این پرتکل ها هم میتونید نگاهی بندازید
SLowDNS DNSST
Hysteria
Tuic 
Ping Tunnel
Mieru
NaïveProxy 
Brook 
RelayBaton 
WireGuard

با تشکر فراوان

**Comment by sina1808** [comment]
کاربر سگارو پروتکل Mieru رو توضیح داده به نظر خوب و کار راه انداز باشه و اگر به هیدیفای اضافه بشه کار رو خیلی راحت میکنه
